adagrams/game.py

```diff
@@ -1,13 +1,182 @@
 from random import randint
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+def convert_letter_pool_to_list(letter_pool):
+    letters_list =[]
+    for letter, count in letter_pool.items():
+        for i in letter * count:
+            letters_list.append(i)
+    return letters_list
 
 def draw_letters():
-    pass
+    draw_letters_list = convert_letter_pool_to_list(LETTER_POOL)
+    result_letters = []
+    counter = 0
+    while counter < 10:
```

Comment:
Since all we need to do is run a loop an arbitrary amount of times, I think that a `for range` loop would be more suitable here. That way we don't have to worry about updating the counter.